contributing.md

```diff
@@ -14,7 +14,7 @@ Here is a sample:
 |----------|------|-------------|
 | `image` | string | **Required** when [using an image](/docs/remote/create-dev-container.md#using-an-image-or-dockerfile). The name of an image in a container registry ([DockerHub](https://hub.docker.com), [GitHub Container Registry](https://docs.github.com/packages/guides/about-github-container-registry), [Azure Container Registry](https://azure.microsoft.com/services/container-registry/)) that VS Code and other `devcontainer.json` supporting services / tools should use to create the dev container. |
 
-- PRs to the [schema](https://github.com/microsoft/vscode/blob/main/extensions/configuration-editing/schemas/devContainer.schema.src.json), i.e code or shell scripts demonstrating approaches for implementation.
+- PRs to the [schema](https://github.com/devcontainers/spec/blob/main/schemas/devContainer.base.schema.json), i.e code or shell scripts demonstrating approaches for implementation.
```

Comment:
My mistake on this one - since this is talking about the spec overall rather than the VS Code specific spec (as the other link points to), I think your first update to `https://github.com/devcontainers/spec/blob/main/schemas/devContainer.base.schema.json` would make sense here!

Reply:
Ah great! I have reverted my last commit